### ISSUE TITLE
[Feature Request]: Subscribe/Unsubscribe Button on channel page. (also show current state)

### ISSUE THREAD

**Issue by nikkdrive18** [yuliskov/SmartTube#4157]
### Describe the feature you'd like to request

Subscribe/Unsubscribe Button on channel page. (also show current state)

### Describe the solution you'd like

Subscribe/Unsubscribe Button on channel page. (also show current state)

### Describe alternatives you've considered

Currently Press and Hold on video before playing > Subscribe/Unsubscribe. (Doesnt show current state)

**Comment by mrcdcs** [comment]
Yes. I think it's quite weird that I have to open the context menu of a video on the Channel page to select Subscribe / Unsubscribe. Would appreciate a button on the page.
